scripts/run.e2e.sh

```diff
@@ -60,7 +60,7 @@ ACK_GINKGO_RC=true ~/go/bin/ginkgo build $extra_build_args ./tests/e2e
 
 if [ "${COVERAGE_MODE:-}" == true ]
 then
-    export GOCOVERDIR=coverage/e2e
+    export GOCOVERDIR=$(PWD)/coverage/e2e
```

Comment:
please check CWD is avalanche-cli repo (parent of the parent of the script)

Reply:
Please check something like

```
if ! [[ "$0" =~ scripts/run.e2e.sh ]]; then
  echo "must be run from repository root"
  exit 1
fi
```

before any other command

Reply:
addressed